roles/kubernetes-apps/external_cloud_controller/digitalocean/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+### Here are DigitalOcean csi crds and localredirect policy crd
+- name: Wait for CRD registration
+  pause:
+    seconds: 30
+  when: inventory_hostname == groups['kube_control_plane'][0]
+
+- name: External Digitalocean Cloud Controller CRDs | Generate Manifests
+  template:
+    src: "{{ item.file }}.j2"
+    dest: "{{ kube_config_dir }}/{{ item.file }}"
+    group: "{{ kube_cert_group }}"
+    mode: 0640
+  with_items:
+    - {name: external-do-csi-crds, file: external-do-csi-crds.yml}
```

Comment:
Those are already installed by `kubernetes-apps/csi_driver/csi_crd` I believe, which you could tweak the install condition in `roles/kubernetes-apps/meta/main.yml`